vsts.yml

```diff
@@ -71,7 +103,7 @@ phases:
      if [ "${COMPONENT}" == "shared_library" ]; then
        script/sccache --stop-server > >(tee -a buildlog.txt) 2> >(tee -a buildlog.txt >&2)
      else
-       if [ "${BUILD_REASON}" == "Schedule" ]; then
+       if [ "Schedule" == "Schedule" ]; then
```

Comment:
...?

Reply:
@nornagon always true! 😄 I'm testing if our libcc tests still run/build properly.  This is a temporary change